src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1990,7 +1990,7 @@
     <value>Starting with .NET 7 the operator '{0}' will throw when overflowing in a checked context. Wrap the expression with an 'unchecked' statement to restore the .NET 6 behavior.</value>
   </data>
   <data name="PreventNumericIntPtrUIntPtrBehavioralChangesTitle" xml:space="preserve">
-    <value>Prevent from behavioral change</value>
+    <value>Prevent behavioral change</value>
```

Comment:
Maybe better to mention that its related to the behavioral changes in IntPtr/UIntPtr, something like
```suggestion
    <value>Prevent numeric IntPtr/UIntPtr behavioral changes</value>
```